go.mod

```diff
@@ -1,47 +1,48 @@
 module github.com/grafana/xk6-client-prometheus-remote
 
-go 1.18
+go 1.19
 
 require (
-	github.com/dop251/goja v0.0.0-20221003171542-5ea1285e6c91
-	github.com/golang/protobuf v1.5.2
+	github.com/dop251/goja v0.0.0-20230531210528-d7324b2d74f7
+	github.com/golang/protobuf v1.5.3
 	github.com/golang/snappy v0.0.4
 	github.com/oxtoacart/bpool v0.0.0-20190530202638-03653db5a59c
```

Comment:
Please run `go mod tidy`. The requirement for `oxtoacart/bpool` is no longer necessary.